crates/torii/grpc/src/server/mod.rs

```diff
@@ -126,6 +127,18 @@ impl DojoWorld {
         self.entities_by_hashed_keys(None, limit, offset).await
     }
 
+    async fn events_all(&self, limit: u32, offset: u32) -> Result<Vec<proto::types::Event>, Error> {
+        let query = r#"
+            SELECT keys, data, transaction_hash
+            FROM events
+            LIMIT ? OFFSET ?
```

Comment:
I think we'd want to order by events id (descending) so results are chronological